pkg/new-ui/v1beta1/frontend/src/app/pages/experiments/experiments.component.html

```diff
@@ -1,5 +1,5 @@
 <lib-namespace-select
-  *ngIf="!env.production"
+  *ngIf="(ns.dashboardConnected$ | async) === dashboardDisconnectedState"
```

Comment:
Since you are not using `env.production` more, do you still need this: https://github.com/kubeflow/katib/tree/d9b4602280748e2fb322bffef95d02dda92664cb/pkg/new-ui/v1beta1/frontend/src/environments ?

Reply:
While with this PR we don't use Angular's [environment mechanism](https://angular.io/guide/build#configure-environment-specific-defaults) anymore I would suggest that we keep these files.

They act as a central place of configuration we could use in the future.

Reply:
Got it, thanks.